docs/src/managed/managedcluster.md

```diff
@@ -0,0 +1,93 @@
+| `OCI_MANAGED_NODE_IMAGE_ID`           |                     | The OCID of the image for the Kubernetes worker nodes. Please read the [doc][node-images] for more details.            |
+| `OCI_MANAGED_NODE_SHAPE `             | VM.Standard.E4.Flex | The [shape][node-images-shapes] of the Kubernetes worker nodes.                                                        |
+| `OCI_MANAGED_NODE_MACHINE_TYPE_OCPUS` | 1                   | The number of OCPUs allocated to the worker node instance.                                                             |
+| `OCI_SSH_KEY`                         |                     | The public SSH key to be added to the Kubernetes nodes. It can be used to login to the node and troubleshoot failures. |
+
+## Pre-Requisites
+
+### Environment Variables
+
+Managed clusters also require the following feature flags set as environment variables before [installing 
+CAPI and CAPOCI components using clusterctl][install-cluster-api].
+
+```bash
+export EXP_MACHINE_POOL=true
+export EXP_OKE=true
+```
+
+### OCI Security Policies
+
+Please read the [doc][oke-policies] and add the necessary policies required for the user group.
+Please add the policies for dynamic groups if instance principal is being used as authentication 
+mechanism. Please read the [doc][install-cluster-api] to know more about authentication mechanisms.
+
+## Workload Cluster Templates
+
+Choose one of the available templates to create your workload clusters from the
+[latest released artifacts][latest-release]. The managed cluster templates is of the
+form `cluster-template-managed-<flavour>`.yaml . The default managed template is
+`cluster-template-managed.yaml`. Please note that the templates provided are to be considered
+as references and can be customized further as the [CAPOCI API Reference][api-reference].
+
+## Supported Kubernetes versions
+The [doc][supported-versions] lists the Kubernetes versions currently supported by OKE.
+
+## Create a new OKE cluster.
+
+The following command will create an OKE cluster using the default template. The created node pool uses 
+[VCN native pod networking][vcn-native-pod-networking].
+
+```bash
+OCI_COMPARTMENT_ID=<compartment-id> \
+OCI_MANAGED_NODE_IMAGE_ID=<ubuntu-custom-image-id> \
+OCI_SSH_KEY=<ssh-key>  \
+KUBERNETES_VERSION=v1.24.1 \
+NAMESPACE=default \
+clusterctl generate cluster <cluster-name>\
+--from cluster-template-managed.yaml | kubectl apply -f -
+```
+
+## Create a new private OKE cluster.
+
+The following command will create an OKE private cluster. In this template, the control plane endpoint subnet is a
+private subnet and the API endpoint is accessible only within the subnet. The created node pool uses
+[VCN native pod networking][vcn-native-pod-networking].
+
+```bash
+OCI_COMPARTMENT_ID=<compartment-id> \
+OCI_MANAGED_NODE_IMAGE_ID=<ubuntu-custom-image-id> \
+OCI_SSH_KEY=<ssh-key>  \
+KUBERNETES_VERSION=v1.24.1 \
```

Comment:
Should we call out the version or use a `<kubernetes-version>` or something. I've just noticed most of our docs have a version. I'm wondering if we should do that or not.